share/gpodder/extensions/subscription_stats.py

```diff
@@ -66,27 +67,21 @@ def add_page(self, notebook, category, channels):
         channelcell.set_property('alignment', Pango.Alignment.LEFT)
         channelcell.set_property('ellipsize', Pango.EllipsizeMode.END)
         channelcolumn = Gtk.TreeViewColumn(_('Podcast'))
```

Comment:
Maybe also use 'Channel' here as column header?

Reply:
Changed to Channel.

Reply:
This is probably something we should deal with soon. The menus, notebook tab, and "All episodes - from all podcasts" use "podcast", while both context menus use "channel". Several extensions use "podcast" in their description, and the youtube-dl extension uses "channel" on its preferences page.

Reply:
Grepping `messages.pot` gives about 50 results for '[Pp]odcast'. I can convert these to 'channel' where applicable, if that's OK?

Reply:
@tpikonen I have no problem changing to channel as I am guilty of using it in places. However, channel really only applies to youtube channels, with most feeds being called podcasts. And the title on gpodder.org is "gPodder: Media aggregator and podcast client". We could start a discussion to rename it to channel, but podcast is probably better.